<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Aaron Brooks</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Personal website for Aaron Brooks, systems biologist in Seattle, WA">
    <meta name="author" content="Aaron Brooks">

    <!-- Le styles -->
    <link href="../css/bootstrap.css" rel="stylesheet">
    <link href="../css/bootstrap-responsive.css" rel="stylesheet">
    <link href="../css/carousel.css" rel="stylesheet">
    <link rel="stylesheet" href="../js/colorbox-master/colorbox.css" />


    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
      <script src="./js/html5shiv.js"></script>
    <![endif]-->

    <!-- Fav and touch icons -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="./ico/apple-touch-icon-144-precomposed.png">
    <link rel="apple-touch-icon-precomposed" sizes="114x114" href="./ico/apple-touch-icon-114-precomposed.png">
      <link rel="apple-touch-icon-precomposed" sizes="72x72" href="./ico/apple-touch-icon-72-precomposed.png">
                    <link rel="apple-touch-icon-precomposed" href="./ico/apple-touch-icon-57-precomposed.png">
                                   <link rel="shortcut icon" href="./ico/favicon.png">
  </head>

  <body>

  <div id="wrap">

    <!-- NAVBAR
    ================================================== -->
     <div class="navbar-wrapper">
      <!-- Wrap the .navbar in .container to center it within the absolutely positioned parent. -->
      <div class="container">

        <div class="navbar navbar-inverse">
          <div class="navbar-inner">
            <!-- Responsive Navbar Part 1: Button for triggering responsive navbar (not covered in tutorial). Include responsive CSS to utilize. -->
            <button type="button" class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="brand" href="../index.html">Aaron Brooks</a>
            <!-- Responsive Navbar Part 2: Place all navbar contents you want collapsed withing .navbar-collapse.collapse. -->
            <div class="nav-collapse collapse">
              <ul class="nav">
                <li><a href="../index.html">Home</a></li>
                <li><a href="../about.html">About</a></li>
                <li><a id="contact" href="#">Say Hi</a></li>
                <li><a href="http://scalefreegan.wordpress.com/">Blog</a></li>
                <!-- Read about Bootstrap dropdowns at http://twitter.github.com/bootstrap/javascript.html#dropdowns -->
                <li class="dropdown">
                  <a href="#" class="dropdown-toggle" data-toggle="dropdown">More <b class="caret"></b></a>
                  <ul class="dropdown-menu">
                    <li><a href="../cv.html">CV</a></li>
                    <li><a href="../links.html">Links</a></li>
                  </ul>
                </li>
              </ul>
            </div><!--/.nav-collapse -->
          </div><!-- /.navbar-inner -->
        </div><!-- /.navbar -->

      </div> <!-- /.container -->
    </div><!-- /.navbar-wrapper -->

    <div class="container marketing">

      <div class="featurette">
            <h2 class="featurette-heading">Adaptation of cells to new environments</h2>
            <p class="lead">The evolutionary success of an organism is a testament to its inherent capacity to keep pace with environmental conditions that change over short and long periods. Mechanisms underlying adaptive processes are being investigated with renewed interest and excitement. This revival is partly fueled by powerful technologies that can probe molecular phenomena at a systems scale. Such studies provide spectacular insight into the mechanisms of adaptation, including rewiring of regulatory networks via natural selection of horizontal gene transfers, gene duplication, deletion, readjustment of kinetic parameters, and myriad other genetic reorganizational events. Here, we discuss advances in prokaryotic systems biology from the perspective of evolutionary principles that have shaped regulatory networks for dynamic adaptation to environmental change.</p>
            <p>
              <a class="btn btn-large btn-alert" href="http://www.ncbi.nlm.nih.gov/pubmed/21197660" target="_blank">Pubmed</a>
              <a class="btn btn-large btn-primary" href="../content/brooksetal_2010.pdf">Download</a>
            </p>
      </div>
      <hr class="featurette-divider">
        <div class="featurette" style="padding-top:30px">
          <a class="fig1" href="#fig1"><img class="featurette-image pull-right" height="500px" width="500px" src="../content/brooksetal_2010/figure1_thumb.png"></a>
          <h3 class="featurette-heading"><strong>Networks</strong> represent the complexity of <span class="muted">gene regulation.</span></h3>
          <p> Networks (or graphs) are a useful way to represent, catalog, and analyze diverse types of biological data. Connectivity patterns among various components of the network (the topology) reveals how the cell operates.</p>
          <a class="fig1" href="#fig1">More</a>
          
        </div>
        <hr class="featurette-divider">
        <div class="featurette" style="padding-top:30px">
          <a class="fig2" href="#fig2"><img class="featurette-image pull-left" height="500px" width="500px" src="../content/brooksetal_2010/figure2_thumb.png"></a>
          <h3 class="featurette-heading"><span class="muted">Molecular</span> changes shape <strong>regulatory evolution</strong>.</h3>
          <p>Rewired regulatory networks lead to dramatic changes in cell behavior. Rather than changing gene content, microbes may adapt by simply altering how they regulate their current gene repertoire.</p>
          <a class="fig2" href="#fig2">More</a>
          
        </div>
        <hr class="featurette-divider">
        <div class="featurette" style="padding-top:30px">
          <a class="fig3" href="#fig3"><img class="featurette-image pull-right" height="500px" width="500px" src="../content/brooksetal_2010/figure3_thumb.png"></a>
          <h3 class="featurette-heading"><strong>Systems</strong> approaches to study <span class="muted">regulatory network evolution</span>.</h3>
          <p>Powerful new technologies are being leveraged to gain insight into how regulatory networks in microbes evolve.</p>
          <a class="fig3" href="#fig3">More</a>
        </div>
    </div>

    <footer>
      <hr class="featurette-divider">
      <div class="container">
        <p class="pull-right"><a href="#">Back to top</a></p>
        <p>&copy; 2013 Aaron Brooks &middot; <a href="#">Privacy</a> &middot; <a href="#">Terms</a></p>
      </div>
    </footer>  

<!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../js/jquery-1.9.1.min.js"></script>
    <script src="../js/bootstrap.js"></script>
    <script src="../js/holder.js"></script>
    <script src="../js/colorbox-master/jquery.colorbox-min.js"></script>
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
      ga('create', 'UA-40493126-1', 'aaron-brooks.org');
      ga('send', 'pageview');
    </script>
    <script>
      $(document).ready(function(){   
                $("#contact").popover({ 
                    trigger:"click",
                    placement:"bottom",
                    title:"Email me:",
                    content:"abrooks AT systemsbiology.org",
                    delay: { show: 0, hide: 1000 }
                    });
                $(".fig1").colorbox({ 
                    html:"<img height='609px' width='792px' src='../content/brooksetal_2010/figure1.png'><p>Networks are a useful formalism to represent, catalog, and analyze biological information. (a) Networks are generic entities that are used to represent many types of biological interactions. The fundamental units of a network (or graph) are nodes and edges. Three types of biological information are commonly represented as a network: transcriptional, metabolic, and protein–protein interactions. The types of data used to build the network and the features that define nodes and edges vary by application. (b) Biological networks share common features. Here, we represent a transcriptional network. Three features commonly define network topology and have important dynamic consequences for the behavior of the network: (1) Hierarchy: transcriptional networks are close to scale-free in the distribution of regulatory connections and exhibit hierarchical arrangement of connections. At the beginning of many biological pathways are few ‘master regulators’ that initiate response to environmental or internal cues. These master regulators propagate information to ‘middle managers’ that have many additional regulatory connections, mostly to ‘lower-tier’ regulators that mediate specific biological functions (2) Modularity: biological networks aggregate pathways and functions into modules, which are defined by groups of genes, regulators, and gene products that are somehow interconnected and interdependent. Here, we denote a transcriptional ‘module’ by a purple ring surrounding the nodes in the module, although we note that biological modules are often composed of a diversity of part types that need not be co-regulated. (3) Motifs: interesting dynamic behaviors of gene circuits are often mediated by particular wiring of the parts, which defines a motif. The members of a particularly well-studied motif, the feed-forward loop, are depicted in this illustration by green circles surrounding the nodes. (c) Biological networks learned from experimental data, such as the Environmental and Gene Regulatory Influence Network (EGRIN) for H. salinarum sp. NRC-1, contain many layers of information that can be mined to aid hypothesis generation. (d) that yield functional and mechanistic insights, such as regulation of copper efflux in Halobacterium.</p>",
                      transition:"elastic",
                      width:"792px",
                      height:"900px",
                      maxHeight:"90%",
                    });
                $(".fig2").colorbox({ 
                    html:"<img height='384px' width='792px' src='../content/brooksetal_2010/figure2.png'><p>Several molecular mechanisms mediate topological changes within GRNs. (a) Duplication of transcription factor (TF) and/or target genes (TG). Duplicated copies of a TF or downstream gene initially share the same interactions as its ancestor. Following duplication, however, either copy can subfunctionalize to contain a subset of those ancestral connections or neofunctionalize by gaining new interactions. Subfunctionalization and neofunctionalization generally occur via random mutations. (b) Mutations can occur in the coding or cis-regulatory sequences of either TFs or TGs. Mutations in the cis-regulatory sequences of a TG only affect interaction with that particular target, whereas mutations in coding and cis regions of TF may affect all downstream interactions. (c) Microbial genomes can be extensively modified by horizontal gene transfers (HGTs). Genomes can horizontally inherit new TF (green circle), TG (yellow circle), or both TF and its target simultaneously (not shown). TFs and TG are depicted in blue and orange circles, respectively. The cis-regulatory regions are denoted by gray boxes attached to the circles.</p>",
                      transition:"elastic",
                      width:"792px",
                      height:"650px",
                      maxHeight:"90%",
                    });
                $(".fig3").colorbox({ 
                    html:"<img height='658px' width='603px' src='../content/brooksetal_2010/figure3.png'><p>Understanding GRN evolution starts with a naturally evolved gene regulatory network (GRN) that has increased fitness in a given environment. Initially, the architecture of the network is unknown (indicated by gray shading). To delineate the connectivity of the network, we perturb it via genetic or environmental alterations and observe the phenotypic response across multiple data types (orange box). The data is analyzed and integrated using network inference tools to build a comprehensive view of the changes to identify architectures of optimal network solutions (blue circle). Many iterations of this cycle for different environments yield to library of optimal network solutions which can be compared with derive evolutionary insights (green circle) and further formulate hypotheses. These hypotheses are tested by construction of rewired GRNs by introducing variations at specific components (gray elipse). Rewired GRNs are selected in different environments and characterized in the next iteration.</p>",
                      transition:"elastic",
                      width:"603px",
                      height:"900px",
                      maxHeight:"90%",
                    });
              })
    </script>
  </body>
</html>